src/components/Common/SearchBox.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState, forwardRef } from 'react';
+import remcalc from 'remcalc';
+import styled, { keyframes } from 'styled-components';
+import is from 'styled-is';
+
+import { Input } from './Forms';
+import Anchor from './Anchor';
+
+const fadeIn = keyframes`
+  from{
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+`;
+
+export const SearchField = styled.section`
+  position: relative;
+  margin-bottom: ${remcalc(36)};
+`;
+
+export const Results = styled.ul`
+  width: 100%;
+  max-height: ${remcalc(200)};
+  list-style-type: none;
+  background: ${props => props.theme.colors.greyBg};
+  position: absolute;
+  left: 0;
+  top: ${remcalc(56)};
```

Comment:
56px is off from our spacing system... shouldn't we use 54px -> theme.space[4] ?

Reply:
will do it